src/lib.rs

```diff
@@ -25,6 +25,7 @@ extern crate rand;
 #[cfg(feature = "rand")]
 mod random_state;
 
+#[cfg(feature = "std")]
 mod seeded_state;
```

Comment:
This is not a correct fix. `FxSeededState` _is_ `no_std` compatible. The change should be specifically to `FxHashMapSeed` and `FxHashSetSeed` which use `std::collections`.

Reply:
Ok so I changed the test to use the hasher instead of the collection